tests/replicate_test.go

```diff
@@ -18,14 +18,16 @@ func TestReplication(t *testing.T) {
 	Convey("orbit-db - Replication", t, FailureHalts, func(c C) {
 		var db1, db2 orbitdb.EventLogStore
 
+		inMemoryPath := ":memory:"
```

Comment:
Instead of choosing in-memory for testing instead of real file; can you just run the whole test suite against the two environments?